src/views/catalog/CreateFromInstanceTypes/state/hooks/useInstanceTypes.ts

```diff
@@ -0,0 +1,24 @@
+import { VirtualMachineClusterInstancetypeModelGroupVersionKind } from '@kubevirt-ui/kubevirt-api/console';
+import { V1beta1VirtualMachineClusterInstancetype } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { isEmpty } from '@kubevirt-utils/utils/utils';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+
+type UseInstanceTypes = (
+  namespaced?: boolean,
```

Comment:
namespaced prop is not needed as these CR are cluster scoped and in all cases namespaced should be false, please remove it